### PR TITLE
Mom 239 more tests

### PR DESCRIPTION
**Pull request by egrimstad**

### CODE REVIEW

File: src/main/java/no/dusken/momus/controller/ArticleController.java [hunk 1]
```diff
@@ -150,55 +105,60 @@ public void getIndesignExport(@PathVariable("id") Long id, HttpServletResponse r
         outStream.close();
     }
 
-    @RequestMapping(value = "/{id}/revisions", method = RequestMethod.GET)
+    @GetMapping("/{id}/revisions")
     public @ResponseBody List<ArticleRevision> getArticleRevisions(@PathVariable("id") Long id) {
```
Comment (eirism) [inline src/main/java/no/dusken/momus/controller/ArticleController.java]: Shouldn't ("id") be dropped from this signature, to be consistent?

Comment (egrimstad) [reply src/main/java/no/dusken/momus/controller/ArticleController.java]: Yes, I agree

File: src/test/java/no/dusken/momus/service/ArticleServiceTest.java [hunk 2]
```diff
@@ -16,14 +16,16 @@
 
 package no.dusken.momus.service;
 
+import com.google.api.services.drive.model.File;
 import no.dusken.momus.model.*;
+import no.dusken.momus.service.drive.GoogleDriveService;
```
Comment (eirism) [inline src/test/java/no/dusken/momus/service/ArticleServiceTest.java]: File and GoogleDriveService seems to be unused

File: src/test/java/no/dusken/momus/service/AbstractServiceTest.java [hunk 3]
```diff
@@ -14,18 +14,43 @@
...
+package no.dusken.momus.service;
 
+import no.dusken.momus.authentication.UserDetailsService;
+import no.dusken.momus.model.Person;
+import org.junit.Before;
 import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
 import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.context.annotation.PropertySource;
```
Comment (eirism) [inline src/test/java/no/dusken/momus/service/AbstractServiceTest.java]: Before and PropertySource seems to be unused.

File: src/main/java/no/dusken/momus/controller/PublicationController.java [hunk 4]
```diff
@@ -24,6 +24,8 @@
 import no.dusken.momus.service.ArticleService;
```
Comment (eirism) [inline src/main/java/no/dusken/momus/controller/PublicationController.java]: Article and ArticleService seems to be unused.

**Pull request merged**